src/autorun.ts

```diff
@@ -24,20 +37,40 @@ export function autorun<T>(fn: Cb<T>) {
     return run<T>(fn).subscribe();
 }
 
+const throwCtxtError = () => { throw new Error('$ or _ can only be called within a run() context'); }
+const errorTracker: $FnWithTrackers = Object.assign({},
```

Comment:
**important:** `Object.assign` probably would not work here: it won't be callable

So the test must be is throwing that it's not a fn, rather than the error above

Reply:
Yes you are right, good catch!

>     Received message:   "context[tracker] is not a function"

I thought I've seen the error, but that must have been from `.weak` or `.normal`. Will fix!